test/query.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const app = require('../src/backend/web/app');
+
+jest.mock('../src/backend/utils/elastic');
+
+describe('Testing query route', () => {
+  test('Testing with valid characters', async () => {
+    const searchTerm = 'I Love Telescope';
+    const res = await request(app).get(`/query?search=${searchTerm}`);
+    expect(res.status).toBe(200);
+  });
+
+  test('Testing with invalid characters', async () => {
+    // searchTerm is 257 chars long
+    const searchTerm =
+      '8l4XOYWZ3SA9aevIozTcEAng3GOSCAiiDARThEkAFn2F2YtBexA3lcg1O38SGSHILQrrNYReKWOC6RM4ZQQIGqZoLSOLlbbYqlfSkIDM83aeGDYW7KU8OSLbIXUIWIF4TINwrjxi453biwyjgYsJeqFx9ORd0EIw3dMwGPWhoMbvTIxUWXV032qgPRmohLbTf8xnMyttPjIOk3rHBXpukWSnkZiKyBMsUniZZnxYPw7yIhfoaS77jIPRUuiQufDdO';
+    const res = await request(app).get(`/query?search=${searchTerm}`);
+    expect(res.status).toBe(400);
```

Comment:
Let's assert that the length is what we want here so it doesn't get changed in follow-up code (not sure what number you are trying to test exactly).  Also, these aren't "invalid characters" are they?  This test seems to be about maximum length.

```
expect(searchTerm.length).toBeGreaterThan(256);
```